exposed-tests/src/test/kotlin/org/jetbrains/exposed/sql/tests/shared/TransactionIsolationTest.kt

```diff
@@ -117,7 +117,7 @@ class TransactionIsolationTest : DatabaseTestsBase() {
     private fun Transaction.assertTransactionIsolationLevel(testDb: TestDB, expected: Int) {
         val (sql, repeatable, committed) = when (testDb) {
             TestDB.POSTGRESQL -> Triple("SHOW TRANSACTION ISOLATION LEVEL", "repeatable read", "read committed")
-            TestDB.MYSQL, TestDB.MARIADB -> Triple("SELECT @@tx_isolation", "REPEATABLE-READ", "READ-COMMITTED")
+            TestDB.MYSQL_V5, TestDB.MARIADB -> Triple("SELECT @@tx_isolation", "REPEATABLE-READ", "READ-COMMITTED")
```

Comment:
This should also work with MySQL8. Rather than exclude, please include fix detailed in [other PR comment](https://github.com/JetBrains/Exposed/pull/1967#discussion_r1459025465):
```kt
private fun Transaction.assertTransactionIsolationLevel(testDb: TestDB, expected: Int) {
    // ...
    TestDB.MYSQL, TestDB.MARIADB -> Triple("SELECT @@transaction_ISOLATION", "REPEATABLE-READ", "READ-COMMITTED")
}
```